src/griffe/agents/nodes.py

```diff
@@ -508,6 +508,7 @@ def _pick_member(self, name: str, member: Any) -> bool:
             and member is not type
             and member is not object
             and id(member) not in self._ids
+            and name in vars(self.obj)
```

Comment:
From https://github.com/mkdocstrings/griffe/issues/153, thanks @tazend! The goal here is to only get members that are not inherited, and rely on the abstract (as in, visitor-inspector agnostic) `Object.inherited_members` property.